ci_config.json

```diff
@@ -126,6 +126,10 @@
       "cpp_std=c++14"
     ]
   },
+  "libobsd": {
+    "_comment": ["Doesn't build with MSVC"],
+    "linux_only": true
```

Comment:
Since we have macOS CI now, this probably should have migrated to the new method `"build_on": { "windows": false }`.

Reply:
I'll do it soon, I happen to have an update that fixes the Windows build in the works, so it seems sensible to do both the migration and the update at once.

Reply:
Ah, in that case we can simply drop the override config entirely I guess?

Reply:
Yes, I worded myself poorly.